packages/react/src/themes/teams/components/Tree/treeTitleStyles.ts

```diff
@@ -0,0 +1,18 @@
+import { ICSSInJSStyle } from '../../../types'
+import { pxToRem } from '../../../../lib'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+
+const treeTitleStyles = {
+  root: ({ variables, theme: { siteVariables } }): ICSSInJSStyle => ({
+    padding: `${pxToRem(1)} 0`,
+    cursor: 'pointer',
+    color: variables.defaultColor,
```

Comment:
```suggestion
    color: v.color,
```

https://github.com/stardust-ui/react/pull/1779/files#r319038491